pkg/utils/exec/exec.go

```diff
@@ -1,4 +1,4 @@
-package k8shandler
+package exec
```

Comment:
we could just keep this as `pkg/utils/exec.go` then it would be package `utils` and we would get it when we do an import of `utils`.
Unless you have a specific reasoning for making it separate?

Reply:
moved